addons/website_sale_mass_mailing/views/templates.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.website.sale.mass.mailing</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website_sale.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//setting[@id='website_checkout_registration']" position="after">
+                <setting id="address_newsletter" string="Newsletter"
```

Comment:
Users that don't have this permission won't be able to access the settings page and therefor denied to change other settings
```suggestion
                <setting 
                    id="address_newsletter" 
                    string="Newsletter" 
                    groups="mass_mailing.group_mass_mailing_user"
```